startProfil.json

```diff
@@ -0,0 +1,17 @@
+{
+  "profils": [
+    {
+      "name": "profil1",
+      "apps": [
+       "First APP to execute",
+       "Second APP to execute"
+      ]
+   },
+   {
+    "name": "profil2",
+    "apps": [
+     "First APP to execute"
+    ]
+ }
+  ]
+}
```

Comment:
Instead of adding an invalid config that won't work out of the box I'd suggest to put this piece of code into the Readme as an example and leave it up to the user to create it. In this case we should also make sure that everything works properly if no `startProfil.json` exists.